app/controllers/Study.scala

```diff
@@ -479,6 +479,19 @@ final class Study(
       }
     }
 
+  def chapterPgnPlainText(id: String, chapterId: String) =
+    Open { implicit ctx =>
+      env.study.api.byIdWithChapter(id, chapterId) flatMap {
+        _.fold(notFound) { case WithChapter(study, chapter) =>
+          CanView(study, ctx.me) {
+            env.study.pgnDump.ofChapter(study, requestPgnFlags(ctx.req))(chapter) map { pgn =>
+              Ok(pgn.toString).as("text/plain")
+            }
+          }(privateUnauthorizedFu(study), privateForbiddenFu(study))
+        }
+      }
+    }
+
```

Comment:
that's a copy paste of `exportPgn`. Why is it necessary?

Reply:
This is how I understand https://lichess.org/forum/lichess-feedback/why-cant-i-copy-the-pgn-of-a-study ... may be I was mislead ...

Reply:
I don't see why you can't use the existing endpoint and need to duplicated it

Reply:
oh I see what you mean .... let me test again

Reply:
yeap you're right